config.go

```diff
@@ -157,6 +158,7 @@ type config struct {
 	ReadMacPath    string `long:"readonlymacaroonpath" description:"Path to write the read-only macaroon for lnd's RPC and REST services if it doesn't exist"`
 	InvoiceMacPath string `long:"invoicemacaroonpath" description:"Path to the invoice-only macaroon for lnd's RPC and REST services if it doesn't exist"`
 	LogDir         string `long:"logdir" description:"Directory to log output."`
+	MaxLogFiles    int64  `long:"maxlogfiles" description:"Maximum logfiles to keep (0 for no rotation)"`
```

Comment:
Maybe just make this `Int`?

Reply:
Maybe also add a comment how big each file will be?

Reply:
Make log file size configurable instead?